modules/twinkleprod.js

```diff
@@ -372,13 +424,17 @@ Twinkle.prod.callback.evaluate = function twinkleprodCallbackEvaluate(e) {
 	Morebits.simpleWindow.setButtonsEnabled( false );
 	Morebits.status.init( form );
 
-	Morebits.wiki.actionCompleted.redirect = mw.config.get('wgPageName');
-	Morebits.wiki.actionCompleted.notice = "Tagging complete";
+	var talk_title = new mw.Title(mw.config.get('wgPageName')).getTalkPage().getPrefixedText();
```

Comment:
`mw.Title` isn't particularly Twinkle-y (although it is used elsewhere), but imo it's a lot cleaner for specifically this sort of thing: getting the talk page name.  `Morebits.wikipedia.namespaces` isn't too unwieldy — nor is `wgNamespaceIds` — but this is just so much simpler looking.